Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Lukas Mayrhofer <lukas.mayrhofer@tum.de>"]
 version = "0.1.0"
 
 [deps]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
```

Comment:
I was going to suggest compat bounds for this but it seems Llama2.jl doesn't have them yet. Probably a longer term issue for now.

Reply:
Actually we won't need CUDA dependency anymore, since KernelAbstraction works with "any" GPU compatible array.